assets/jsx/bagger.jsx

```diff
@@ -19,15 +19,11 @@ class Bagger extends React.Component {
     componentDidMount() {
         const { dispatch } = this.props
         const b = document.getElementById('bagger');
-        if (b.dataset.accessKeyId !== undefined) { // TODO: any? all? add fine grain actions?
-            const dataset = b.dataset
-            dispatch(BagActions.updateConfig(
-                dataset.accessKeyId,
-                dataset.secretAccessKey,
-                dataset.bucket,
-                dataset.region,
-                dataset.keyPrefix
-                ))
+        const dataset = b.dataset
+        const args = [dataset.accessKeyId, dataset.secretAccessKey, dataset.bucket, dataset.region,
+                        dataset.keyPrefix]
```

Comment:
We're not currently using esformatter – and at a quick glance it doesn't look like it supports jsx. We also need something that'll support the babel we're currently speaking. For my editor I'm using the language-babel package and it seems to be doing a good job on formatting with the exception of this multiple line example